man/udev_device_has_tag.xml

```diff
@@ -36,6 +38,18 @@
     <funcsynopsis>
       <funcsynopsisinfo>#include &lt;libudev.h&gt;</funcsynopsisinfo>
 
+      <funcprototype>
+        <funcdef>int <function>udev_device_has_tag</function></funcdef>
+        <paramdef>struct udev_device *<parameter>udev_device</parameter></paramdef>
+        <paramdef>const char *<parameter>tag</parameter></paramdef>
+      </funcprototype>
+
+      <funcprototype>
+        <funcdef>int <function>udev_device_has_current_tag</function></funcdef>
+        <paramdef>struct udev_device *<parameter>udev_device</parameter></paramdef>
+        <paramdef>const char *<parameter>tag</parameter></paramdef>
+      </funcprototype>
```

Comment:
Hmmm, sd_device_* is the more modern API, and we should document that first. Should be mostly a copy-paste affair.

Reply:
sd-device API is entirely undocumented so far. Of course we should document that, but I didn#t want to be the one doing that with this PR....